src/int.rs

```diff
@@ -278,10 +285,102 @@ pub trait PrimInt:
     /// assert_eq!(2i32.pow(4), 16);
     /// ```
     fn pow(self, exp: u32) -> Self;
+
+    /// Return the memory representation of this integer as a byte array in big-endian byte order.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use num_traits::PrimInt;
+    ///
+    /// let bytes = 0x12345678u32.to_be_bytes();
+    /// assert_eq!(bytes, [0x12, 0x34, 0x56, 0x78]);
+    /// ```
+    fn to_be_bytes(self) -> Self::Bytes;
```

Comment:
It's a breaking change to add non-defaulted methods to an existing trait, and there's no reasonable default we could provide here.

Instead, I suggest adding an new trait for this new functionality.

Reply:
agree, I added `IntToFromBytes` to `src/ops/bytes.rs`